packages/paste-website/src/pages/patterns/pricing/index.mdx

```diff
@@ -0,0 +1,390 @@
+          </Box>
+          <Box marginTop="space30" display="flex" columnGap="space20" alignItems="flex-end">
+            <Heading as="h3" variant="heading10" marginBottom="space0">
+              $0
+            </Heading>
+            <Paragraph marginBottom="space0">Per month</Paragraph>
+          </Box>
+        </Box>
+        <Box marginY="space50">
+          <Separator orientation="horizontal" />
+        </Box>
+        <Box display="flex" flexDirection="column" rowGap="space50">
+          {["$15 credit", "Testing with upto 5 verified recipients", "Limited configurations"].map((item: string) => (
+            <Box key={item} display="flex" columnGap="space20" alignItems="center">
+              <SelectedIcon decorative color="colorTextIconSuccess" />
+              <Paragraph marginBottom="space0">{item}</Paragraph>
+            </Box>
+          ))}
+        </Box>
+      </VisualPickerRadio>
+      {...}
+    </VisualPickerRadioGroup>
+  )
+}`}
+/>
+
+### Table
+
+Consider using a table when there are 5 or more options for users to choose from, or more comparison is necessary between any number of plans.
+
+<StoryPreview
+  title="Table example"
+  storyID="website-pricingexamples--table-example"
+  height="600px"
+  code={`
+export const TableExample = (): React.ReactNode => {
+  return (
+    <Table>
+      <THead>
+        <Tr verticalAlign="top">
+          <Th> </Th>
+          <Th>
+            <Box display="flex" flexDirection="column" rowGap="space20">
+              Developer
+              <Badge as="span" variant="neutral">
+                Current plan
+              </Badge>
+            </Box>
+          </Th>
+          <Th>Production</Th>
+          <Th>
+            <Box display="flex" flexDirection="column" rowGap="space20">
+              Business
+              <Badge as="span" variant="success">
+                Recommended
+              </Badge>
+            </Box>
+          </Th>
+          <Th>Personalized</Th>
+        </Tr>
+      </THead>
+      <TBody>
+        <Tr>
+          <Td>Price per month</Td>
+          <Td>Included</Td>
+          <Td>
+            4% of monthly spend <DetailText marginTop="space0">or $250.00 minimum</DetailText>
+          </Td>
+          <Td>
+            6% of monthly spend <DetailText marginTop="space0">or $1500.00 minimum</DetailText>
+          </Td>
+          <Td>
+            8% of monthly spend <DetailText marginTop="space0">or $5000.00 minimum</DetailText>
+          </Td>
+        </Tr>
+        <Tr>
+          <Td>Web support</Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconSuccess" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconSuccess" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconSuccess" />
+          </Td>
+          <Td>
+            <SelectedIcon decorative color="colorTextIconSuccess" />
+          </Td>
+        </Tr>
```

Comment:
nit(non-blocking): the figma uses a different token but the actual RGB values are the same

```suggestion
          <Td>
            <SelectedIcon decorative color="colorTextIconAvailable" />
          </Td>
          <Td>
            <SelectedIcon decorative color="colorTextIconAvailable" />
          </Td>
          <Td>
            <SelectedIcon decorative color="colorTextIconAvailable" />
          </Td>
          <Td>
            <SelectedIcon decorative color="colorTextIconAvailable" />
          </Td>
        </Tr>
```